documentation/tutorials/batch.md

```diff
@@ -0,0 +1,186 @@
+
+real    0m24.051s
+user    0m5.258s
+sys     0m0.538s</pre>
+
+This shows that appyling the migration without batching took 24 seconds.
+
+Running `flyway info` will now show that both migrations have been run:
+
+<pre class="console">Database: jdbc:postgresql://instances.spawn.cc:31585/ (PostgreSQL 11.0)
+
++-----------+---------+---------------------+------+---------------------+---------+----------+
+| Category  | Version | Description         | Type | Installed On        | State   | Undoable |
++-----------+---------+---------------------+------+---------------------+---------+----------+
+| Versioned | 1       | Create person table | SQL  | 2017-12-17 19:57:28 | Success | No       |
+| Versioned | 2       | Insert data         | SQL  | 2017-12-17 20:01:13 | Success | No       |
++-----------+---------+---------------------+------+---------------------+---------+----------+</pre>
+
+## Apply the script using batching
+
+We'll revert the database back to the state before we applied the `./sql/V2__insert-data.sql` migration:
+
+```bash
+$ spawnctl reset data-container flyway-container
+```
+
+Rerunning `flyway info` once again shows that only the first migration has been applied:
+
+<pre class="console">Database: jdbc:postgresql://instances.spawn.cc:31585/ (PostgreSQL 11.0)
+
++-----------+---------+---------------------+------+---------------------+---------+----------+
+| Category  | Version | Description         | Type | Installed On        | State   | Undoable |
++-----------+---------+---------------------+------+---------------------+---------+----------+
+| Versioned | 1       | Create person table | SQL  | 2017-12-17 19:57:28 | Success | No       |
+| Versioned | 2       | Insert data         | SQL  |                     | Pending | No       |
++-----------+---------+---------------------+------+---------------------+---------+----------+</pre>
+
+Now edit the `/flyway.conf` file to enable batching by appending this line to the end of the file:
+
+```properties
+flyway.batch=true
+```
+
+And rerun `flyway migrate`:
+
+```bash
+$ time flyway migrate
+```
+
+<pre class="console">
+Flyway Teams Edition 7.11.2 by Redgate
+Database: jdbc:postgresql://instances.spawn.cc:32014/ (PostgreSQL 11.0)
+----------------------------------------
+Flyway Teams features are enabled by default for the next 27 days. Learn more at https://flywaydb.org?ref=v7.11.2_teams
+----------------------------------------
+Successfully validated 2 migrations (execution time 00:00.089s)
+Current version of schema "public": 1
+Migrating schema "public" to version "2 - insert data"
+Successfully applied 1 migration to schema "public", now at version v2 (execution time 00:01.005s)
+
+real    0m4.814s
+user    0m4.541s
+sys     0m0.399s</pre>
+
+With batching enabled the migration only took ~5 seconds to run.
+
+## Summary
+
+In this brief tutorial we saw how to:
+* use flyway batch to batch migration scripts for better performance.
```

Comment:
Could the first instance of the word `batch` here link to [this](https://flywaydb.org/documentation/configuration/parameters/batch) page

Reply:
caa4db38386be82060ea8186c2d162f6180485cf